src/common_rl/from_env.jl

```diff
@@ -0,0 +1,215 @@
+abstract type AbstractRLEnvMDP{S, A} <: MDP{S, A} end
+abstract type AbstractRLEnvPOMDP{S, A, O} <: POMDP{S, A, O} end
+
+const AbstractRLEnvProblem = Union{AbstractRLEnvMDP, AbstractRLEnvPOMDP}
+
+POMDPs.actions(m::AbstractRLEnvProblem) = RL.actions(m.env)
+POMDPs.discount(m::AbstractRLEnvProblem) = m.discount
+
+function POMDPs.actions(m::AbstractRLEnvProblem, s)
+    if provided(RL.valid_actions, m.env)
+        RL.setstate!(m.env, s)
```

Comment:
Should we reset the state of `m.env` what it was before in case it's stateful?

Reply:
Yes! Excellent point!

Reply:
will fix!